curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-102-triangle-containment.english.md

```diff
@@ -12,7 +12,7 @@ Consider the following two triangles:
 A(-340,495), B(-153,-910), C(835,-947)
 X(-175,41), Y(-421,-714), Z(574,-645)
 It can be verified that triangle ABC contains the origin, whereas triangle XYZ does not.
-Using triangles.txt (right click and 'Save Link/Target As...'), a 27K text file containing the co-ordinates of one thousand "random" triangles, find the number of triangles for which the interior contains the origin.
```

Comment:
This corpus used both spellings. The majority of the world and this corpus favor no `-`.